app/views/items/show.html.erb

```diff
@@ -2,3 +2,36 @@
 <h2><%= @item.subtitle %></h2>
 <h2><%= @item.price %></h2>
 <img src="<%= @item.picture_url %>" alt="<%= @item.title %>" />
+
+<h4>Reviews</h4>
+<ul>
+  <% if @reviews.empty? %>
+    <li>No reviews yet!</li>
+  <% else %>
+    <% @reviews.each do |review| %>
+      <li>Rating: <%= review.rating %></li>
+      <ul>
+        <li><%= review.description %></li>
+      </ul>
+    <% end %>
+  <% end %>
+</ul>
+
+<h4>We'd love your review of the <%= @item.title %>!</h4>
+
+<%= form_for [@item, @review] do |f| %>
+  <%= f.label :rating, "Rating:" %>
+  <%= f.select :rating, options_for_select([
+    ["Pick a Rating", 0],
```

Comment:
Recommend defining this outside of the view. You could put this as a method on the `Review` model as a constant, and then pass it in as an instance variable from your controller (`@rating_options = Review::RATING_OPTIONS`)